<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ACMainWindow</class>
 <widget class="QMainWindow" name="ACMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1017</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AgeCreator</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1017</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionFileNew"/>
    <addaction name="actionFileOpen"/>
    <addaction name="separator"/>
    <addaction name="actionFileSave"/>
    <addaction name="actionFileSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionFileClose"/>
    <addaction name="actionFileQuit"/>
   </widget>
   <widget class="QMenu" name="menuAge">
    <property name="title">
     <string>&amp;Age</string>
    </property>
    <addaction name="actionAgeImport"/>
    <addaction name="actionAgeExport"/>
    <addaction name="separator"/>
    <addaction name="actionAgeProperties"/>
   </widget>
   <widget class="QMenu" name="menuWindows">
    <property name="title">
     <string>Windows</string>
    </property>
    <addaction name="actionToggleObjectList"/>
    <addaction name="actionToggleObjectProperties"/>
    <addaction name="actionToggle3DPreview"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAboutAgeCreator"/>
    <addaction name="actionAboutHSPlasma"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAge"/>
   <addaction name="menuWindows"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Age Objects</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="ACAgeTreeView" name="sceneTreeView"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="buttonAddLayer">
         <property name="toolTip">
          <string>Add Layer...</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="buttonAddObject">
         <property name="toolTip">
          <string>Add Object...</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="popupMode">
          <enum>QToolButton::InstantPopup</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="buttonDelObject">
         <property name="toolTip">
          <string>Delete Object...</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>3D Preview</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="ACGLWidget" name="preview3d" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionAgeImport">
   <property name="text">
    <string>&amp;Import</string>
   </property>
  </action>
  <action name="actionAgeExport">
   <property name="text">
    <string>&amp;Export</string>
   </property>
  </action>
  <action name="actionFileNew">
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="actionFileOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionFileSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionFileSaveAs">
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
  </action>
  <action name="actionFileClose">
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="actionFileQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionAgeProperties">
   <property name="text">
    <string>Properties...</string>
   </property>
  </action>
  <action name="actionToggleObjectList">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Object List</string>
   </property>
  </action>
  <action name="actionToggle3DPreview">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>3D Preview</string>
   </property>
  </action>
  <action name="actionToggleObjectProperties">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Object Properties</string>
   </property>
  </action>
  <action name="actionAboutAgeCreator">
   <property name="text">
    <string>About Age Creator</string>
   </property>
  </action>
  <action name="actionAboutHSPlasma">
   <property name="text">
    <string>About HSPlasma</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ACGLWidget</class>
   <extends>QWidget</extends>
   <header>ACGLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ACAgeTreeView</class>
   <extends>QTreeView</extends>
   <header>ACAgeTreeView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionAgeExport</sender>
   <signal>triggered()</signal>
   <receiver>ACMainWindow</receiver>
   <slot>exportAge()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAgeImport</sender>
   <signal>triggered()</signal>
   <receiver>ACMainWindow</receiver>
   <slot>importAge()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFileClose</sender>
   <signal>triggered()</signal>
   <receiver>ACMainWindow</receiver>
   <slot>closeAge()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFileNew</sender>
   <signal>triggered()</signal>
   <receiver>ACMainWindow</receiver>
   <slot>newAge()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFileQuit</sender>
   <signal>triggered()</signal>
   <receiver>ACMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionToggleObjectList</sender>
   <signal>toggled(bool)</signal>
   <receiver>dockWidget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionToggle3DPreview</sender>
   <signal>toggled(bool)</signal>
   <receiver>dockWidget_2</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>888</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dockWidget</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>actionToggleObjectList</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dockWidget_2</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>actionToggle3DPreview</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>888</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAboutAgeCreator</sender>
   <signal>triggered()</signal>
   <receiver>ACMainWindow</receiver>
   <slot>about()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>508</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAboutQt</sender>
   <signal>triggered()</signal>
   <receiver>ACMainWindow</receiver>
   <slot>aboutQt()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>508</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAboutHSPlasma</sender>
   <signal>triggered()</signal>
   <receiver>ACMainWindow</receiver>
   <slot>aboutHSPlasma()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>508</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>newAge()</slot>
  <slot>closeAge()</slot>
  <slot>importAge()</slot>
  <slot>exportAge()</slot>
  <slot>about()</slot>
  <slot>aboutQt()</slot>
  <slot>aboutHSPlasma()</slot>
 </slots>
</ui>
